cmake/CMakeLists.txt

```diff
@@ -395,6 +395,7 @@ endif()
 pkg_depends(ML-IAP ML-SNAP)
 pkg_depends(MPIIO MPI)
 pkg_depends(ATC MANYBODY)
+pkg_depends(AMOEBA KSPACE)
```

Comment:
This is redundant. The same dependency is already listed a few lines below.

Reply:
Removed this redundant line.